docs/track.rst

```diff
@@ -122,6 +123,10 @@ In the ``operations`` section you describe which operations are available for th
 
 In the ``schedule`` section you describe the workload for the benchmark, for example index with two clients at maximum throughput while searching with another two clients with ten operations per second. The schedule either uses the operations defined in the ``operations`` block or defines the operations to execute inline.
 
+In the ``challenges`` section you describe more than one set of operations, in the event your track needs to test than one set of scenarios. If only one set of operations is needed, they can be specified in ``schedule`` instead.
+
+Creating a track does not require all of the above sections to be used. Tracks that are used against existing data may only rely on querying ``challenges`` and can omit the ``indices``, ``templates``, and ``corpora`` sections.
```

Comment:
As `challenges` are optional, I think we shouldn't reference it in the query scenario; we could just say `may only rely on querying operations`?